libraries/AP_HAL/board/sitl.h

```diff
@@ -78,3 +78,7 @@
 #ifndef HAL_CAN_DRIVER_DEFAULT
 #define HAL_CAN_DRIVER_DEFAULT 0
 #endif
+
+#ifndef AP_UART_MONITOR_ENABLED
+#define AP_UART_MONITOR_ENABLED 1
```

Comment:
should this be limited by BOARD_FLASH_SIZE 256

Reply:
we always have more than 256k in SITL :-)